rnd/autogpt_server/autogpt_server/schema.prisma

```diff
@@ -0,0 +1,124 @@
+datasource db {
+  provider = "sqlite"
+  url      = "file:./database.db"
+}
+
+generator client {
+  provider             = "prisma-client-py"
+  recursive_type_depth = 5
+  interface            = "asyncio"
+}
+
+// This model describes the Agent Graph/Flow (Multi Agent System).
+model AgentGraph {
```

Comment:
I prefer `AgentTemplate`

Reply:
Also I feel that maybe this should have a link to the triggers that can start the process on it.

Reply:
I disagree with `AgentTemplate`, it sounds like a template you can reuse and not like a graph/flow of nodes.
Link to trigger is good.

Reply:
Thats exactly it, it is something that can be reused. 

Given it is something that can be reused, we want the name to make sense to end users without needing to have an end user name overlay at the end.

Calling it a graph or a flow to me is confusing. More people have an intuitive understanding of what templates are as a concept.

Reply:
Since we already call this node, I guess it's very logical to call the set of the node as graph. We can create template and re-use stuff once we have added user-functionality.